src/main/java/org/mule/tools/apikit/model/MuleDomain.java

```diff
@@ -6,21 +6,20 @@
  */
 package org.mule.tools.apikit.model;
 
-import org.jdom2.Document;
-import org.jdom2.input.SAXBuilder;
-import org.mule.tools.apikit.input.parsers.HttpListenerConfigParser;
-
 import java.io.InputStream;
 import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
 
+import org.jdom2.Document;
+import org.jdom2.input.SAXBuilder;
+import org.mule.tools.apikit.input.parsers.HttpListenerConfigParser;
+
 public class MuleDomain implements NamedContent, WithConfigs {
 
-  private InputStream content;
   private List<HttpListenerConfig> configurations;
 
-  private MuleDomain(InputStream content, List<HttpListenerConfig> configurations) {
-    this.content = content;
+  MuleDomain(List<HttpListenerConfig> configurations) {
```

Comment:
why removing the constructor?

Reply:
I removed the access modifier to be able to create an instance from the factory